modules/account-lib/src/coin/dot/transactionBuilder.ts

```diff
@@ -89,17 +90,17 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
   /**
    * The number of the checkpoint block after which the transaction is valid
    *
-   * @param {number} blockNumber
+   * @param validityWindow
```

Comment:
nit: though it is a lowercased word but this is actually a type, may want to wrap that as `{validityWindow}`